utils/uftrace-script.h

```diff
@@ -0,0 +1,54 @@
+#ifndef UFTRACE_UFTRACE_SCRIPT_H
+#define UFTRACE_UFTRACE_SCRIPT_H
+
+#ifndef __cplusplus
+#include <stdint.h>
+#include <stdbool.h>
+#include "utils/list.h"
+#else
+#include <cstdint>
+#include <cstdbool>
+#endif
+
+/* informantion passed during initialization */
+#define SCRIPT_INFO_MAJOR_VERSION 1
+#define SCRIPT_INFO_MINOR_VERSION 0
+struct script_info {
+	struct si_version {
+		uint8_t major;
+		uint8_t minor;
+	} ver;
+	char			*name;
+	char			*version;
+	bool			record;
```

Comment:
Yes, that's also possible. I also thought that it's wasteful to use a bool type here.

Reply:
> Maybe we can have 2 integers as bitmasks for them. One indicates current status - only the record bit is available for now.

For now, I've just made the existing `record` variable can be disabled by the given native script. If it's disabled by script, then uftrace do not actually record the data.

> The other can control the behavior like disable, or skip arguments. I think we should keep original behavior unless script changes the bitmasks.

I don't see any use case for disabling argument record because we can already do this with uftrace command. I will just leave it as is, but can bump up the script version when it's needed.